### ISSUE TITLE
Create and test proper validation set

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/CancerDetection#9]
We need to create a proper validation set by removing sets of images that came from the same original slide picture. After creating the validation set we should train a few networks on it and compare the validation AUC score to what we get on the leaderboard. 

When we increase validation score, does our LB usually improve?

**Comment by JoshVarty** [comment]
### All transforms on Resnet50
| Epochs  | Valid | Test  | 
|---|---|---|
| 10 | 0.935 | 0.9612 |
| 20 | 0.938 | 0.9624 |
| 30 | 0.934 | 0.9628 |
| 40 | 0.940 | 0.9640 |
| 50 | 0.938 | 0.9637 |
| 60 | 0.941 | 0.9654 |
| 70 | 0.937 | 0.9645 |
| 100* | 0.939 | 0.9686 |
| 160* | 0.945 | 0.9690 |

\* Trained using `.fit()`

**Comment by JoshVarty** [comment]
### All transforms on Resnet101

| Epochs  | Valid | Test  | 
|---|---|---|
| 10 | 0.884 | 0.9412 |
| 20 | 0.898 | 0.9440 |
| 30 | 0.901 | 0.9469 |
| 40 | 0.901 | 0.9492 |
| 50 | 0.904 | 0.9496 |
| 60 | 0.908 | 0.9522 |
| 70 | 0.916 | 0.9530 |
| 100* | 0.924 | 0.9555 |
| 160* | 0.937 | 0.9599 |

**Comment by JoshVarty** [comment]
### All transforms on Resnet50 (random validation set)
| Epochs  | Valid | Test  | 
|---|---|---|
| 10 | 0.986 | 0.9612 |
| 20 | 0.987 | 0.9604 |
| 30 | 0.989 | 0.9603 |
| 40 | 0.990 | 0.9601 |
| 50 | 0.990 | 0.9600 |
| 60 | 0.991 | 0.9611 |
| 70 | 0.992 | 0.9622 |
| 100* | 0.992 | 0.9592 |
| 160* | 0.994 | 0.9578  |

\* Trained using `.fit()`
